### ISSUE TITLE
No lower-case for menu actions

### ISSUE THREAD

**Issue by rugk** [ligi/PassAndroid#116]
In the German version you currently have menus/buttons/… called "löschen" or similar. However it looks ugly as it is best practise to just capitalize all menu entries and similar stuff even if it would be written with a lower-case letter in German.

So please change the buttons, so that the example e.g. looks like "Löschen".

**Comment by ligi** [comment]
Thanks for the suggestion - I usually use english - Just saw you joined the translators - so I am assuming you are about to do these changes?

**Comment by rugk** [comment]
Yes, but I also saw that the English version has the same issue. It's also lower-case and this is bad practise in English too. So rather change it globally.
You can't blame the translators if the original string is bad…

**Comment by pataquets** [comment]
Same for Spanish and Catalan
